projects/core/src/lib/talks/talks-page/talks-page.component.ts

```diff
@@ -0,0 +1,107 @@
+import { Component, OnInit } from "@angular/core";
+import { Talk } from "../models/talk";
+
+@Component({
+	selector: "jsmu-talks-page",
+	templateUrl: "./talks-page.component.html",
+	styleUrls: ["./talks-page.component.scss"]
+})
+export class TalksPageComponent implements OnInit {
+	sortedTalks: Talk[];
+	talks: Talk[] = [
+		{
+			lessonId: "2332",
+			result: "Discussion of features",
+			inProgress: true,
+			expertId: "Maksim",
```

Comment:
Expert id is name?

Reply:
It doesn't matter for mocks, BTW sometimes I do the same in my projects :))